internal/controller/postgrescluster/pgbouncer_test.go

```diff
@@ -175,7 +175,8 @@ ownerReferences:
 		// Defaults to ClusterIP.
 		assert.Equal(t, service.Spec.Type, corev1.ServiceTypeClusterIP)
 		assert.Assert(t, marshalMatches(service.Spec.Ports, `
-- name: pgbouncer
+- appProtocol: tcp
+  name: pgbouncer
```

Comment:
📝 PgBouncer is speaking the PostgreSQL [protocol](https://www.postgresql.org/docs/current/protocol.html), so it should have the same `appProtocol` as the primary and replica Services.

Reply:
yep